### PR TITLE
Remove "Become Psionic" Objective 

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Syndicate Agents will no longer receive the "Become Psionic" objective. 

## Why / Balance
This objective has always sucked. It doesn't encourage our antagonists to be antagonistic, and contributes to stale rounds. This will require review from Fox, but I'm pretty sure we have achieved consensus that this should go. 

## Technical details
I commented out the bit that I think weighs the objectives, but left basically everything else about this Objective- should we wish to re-add it. 

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Shouldn't break anything! I copied what was done with the golem objective :)

**Changelog**
:cl:
- tweak: Syndicate Agents will no longer receive the "Become Psionic" objective.

### CODE REVIEW

**Comment by deltanedas** [comment]
needs to be removed from the group too or it will fuck up traitor assignment

**Review by Colin-Tel — APPROVED** [review]
My understanding of this objective’ presence is that it was a way to “tie in” the psionic/glimmer content to traitor objectives. The difference between this one and “have glimmer above 500” is that this objective is a relatively easy one that’s just a little tedious sometimes.

Totally fine with this change.

**Comment by therealDLondon** [comment]
Requires review from Fox?
Consider it reviewed and accepted

**Comment by DangerRevolution** [comment]
[Resources/Prototypes/Nyanotrasen/Objectives/traitor.yml](https://github.com/DeltaV-Station/Delta-v/blob/e3d4adba0e485d85c2740b32d9d1bb25593e31fc/Resources/Prototypes/Nyanotrasen/Objectives/traitor.yml#L13)

this needs to be commented out too from what i know

**Comment by DangerRevolution** [comment]
Up to @NullWanderer but IIRC the standard here would just be deleting all these files & lines that are associated with this objective instead of commenting it out. At the same time though, the same standard would consider this "deleting content" so um. might be worth commenting everything associated out so someone can come fix this down the line

**Comment by Lyndomen** [comment]
> [Resources/Prototypes/Nyanotrasen/Objectives/traitor.yml](https://github.com/DeltaV-Station/Delta-v/blob/e3d4adba0e485d85c2740b32d9d1bb25593e31fc/Resources/Prototypes/Nyanotrasen/Objectives/traitor.yml#L13)
> 
> this needs to be commented out too from what i know

I think my first commit does this, I couldn't find any other spot with that "BecomePsionicObjective" ID. Not really sure though, thanks for checking.

**Comment by rosieposieeee** [comment]
> might be worth commenting everything associated out so someone can come fix this down the line

it was going to be removed in the psionic rework anyway iirc, so you probably dont need to bother. itll still be in the repo history

**Comment by deltanedas** [comment]
not only is this a non objective, if you drink loto under 250 glimmer it can do a bad rng roll and permanently lock you out of rolling it again  probably requiring you get yourself cloned to try again

**Comment by Squishy77** [comment]
> not only is this a non objective, if you drink loto under 250 glimmer it can do a bad rng roll and permanently lock you out of rolling it again probably requiring you get yourself cloned to try again



> not only is this a non objective, if you drink loto under 250 glimmer it can do a bad rng roll and permanently lock you out of rolling it again probably requiring you get yourself cloned to try again

Well your not locked out, pysonic shock can allow you to roll again. But I do agree that it is a bad objective

**Pull request merged**
